packages/cos-config/cos-config

```diff
@@ -27,7 +27,7 @@
 
 # This is the repository that hosts the signature files used by cosign and luet-cosign plugin during upgrade/deploy to
 # check the artifact signatures
-#COSIGN_REPOSITORY="raccos/releases-green"
+#COSIGN_REPOSITORY="raccos/releases-teal"
```

Comment:
Note after merging #1307 this is no longer used. We should consider changing elemental defaults to use teal.

Reply:
wait, we pick the repo now from green directly? liek a default? argh, I migth have not see that in your PR.

In any case, quay.io now supports cosing signatures, so maybe it would be good to just drop this completely from everywhere....